README.md

```diff
@@ -161,6 +161,12 @@ expect(context.component()).toBeInstanceOf(MyComponent);
 ### `FindWrapper#contains(vdom)`
 Searches for any children matching the vdom or text passed.
 
+### `FindWrapper#childAt(index)`
+Returns `FindWrapper` with child at given index.
```

Comment:
Might be worth saying it's effectively `.children().at(index)`

Reply:
Done